arkane/data/species/supporting_information.csv

```diff
@@ -0,0 +1,2 @@
+Label,Symmetry Number,Number of optical isomers,Symmetry Group,Rotational constant (cm-1),"Calculated Frequencies (unscaled and prior to projection, cm^-1)",Electronic energy (J/mol),"E0 (electronic energy + ZPE, J/mol)",E0 with atom and bond corrections (J/mol),Atom XYZ coordinates (angstrom),T1 diagnostic,D1 diagnostic

```

Comment:
What is the purpose of this commit?

Reply:
The supporting information file is the only way to get information like unscaled frequencies, electronic energy, and T1 diagnostic easily from an Arkane job after the fact